test/test_distributed.py

```diff
@@ -1718,6 +1718,18 @@ def test_DistributedDataParallelCPU(self):
         )
         self._barrier()
 
+    @unittest.skipIf(BACKEND != 'nccl' and BACKEND != 'gloo',
+                     "Only Nccl & Gloo backend support DistributedDataParallel")
+    @skip_if_no_cuda_distributed
+    @skip_if_no_gpu
+    def test_DistributedDataParallel_python(self):
```

Comment:
Can you rename this to be more descriptive of what is being tested?

E.g. `test_DistributedDataParallel_requires_grad`.